Assets/Countly/Plugins/CountlySDK/Countly.cs

```diff
@@ -360,7 +359,7 @@ private void CreateListOfIBaseService()
             _listeners.Clear();
 
             _listeners.Add(_push);
-            _listeners.Add(Views);
+            _listeners.Add((ViewCountlyService) Views);
```

Comment:
why this needs cast but others not

Reply:
I assume in the long term, all of those will require a cast. Since "Views" is now an interface, this cast is required.